src/services/sync-component-with-file/sync-component.test.ts

```diff
@@ -66,176 +61,25 @@ describe('syncComponent', () => {
     mockGetProjectLabels.mockResolvedValue(MOCK_GET_PROJECT_LABELS);
   });
 
-  it('creates external alias for component in case data manager is not present and external alias was not created previously for the project', async () => {
+  it('should call syncComponentByExternal Alias', async () => {
     const event = generatePushEvent();
     const compassYaml = getMockedCompassYaml();
-    const component = getMockedComponent({
```

Comment:
most of these tests were moved to the SDK, and could be removed. The ones I kept were about the label it adds, the link it adds, and the catch of a failure still reports an error. Then I added one test to make sure it calls syncComponentWithFile with the right params.